### PR TITLE
Fixed cors origin port issue

### PR DESCRIPTION
**Pull request by lukyanov**
Repeating this:
https://git.teknorota.com/yekmyk/leptus/commit/b417c7b0693125522a5f9e238ad99550d4adf293

Fixes https://github.com/sinasamavati/leptus/issues/55

### CODE REVIEW

**Comment by tommyjcarpenter** [comment]
Does this fix it? What does cross_domains callback value need to be?

I'm currently using:
```
cross_domains(_Route, _Req, State) ->
    {['_'], State}.
```

No port works:
```
curl -H "Origin: http://mydomain.com" --verbose localhost:7777/application/
*   Trying ::1...
* TCP_NODELAY set
* Connection failed
* connect to ::1 port 7777 failed: Connection refused
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 7777 (#0)
> GET /application/ HTTP/1.1
> Host: localhost:7777
> User-Agent: curl/7.54.0
> Accept: */*
> Origin: http://mydomain.com
>
< HTTP/1.1 200 OK
< connection: keep-alive
< server: Cowboy
< date: Tue, 12 Sep 2017 15:20:51 GMT
< content-length: 2
< content-type: application/json
< access-control-allow-origin: http://mydomain.com
<
* Connection #0 to host localhost left intact
[]⏎      
```

Port does not work:
```                                                                                                                                                      tommy  ~  curl -H "Origin: http://mydomain.com:8000" --verbose localhost:7777/application/
*   Trying ::1...
* TCP_NODELAY set
* Connection failed
* connect to ::1 port 7777 failed: Connection refused
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 7777 (#0)
> GET /application/ HTTP/1.1
> Host: localhost:7777
> User-Agent: curl/7.54.0
> Accept: */*
> Origin: http://mydomain.com:8000
>
< HTTP/1.1 500 Internal Server Error
< connection: keep-alive
< server: Cowboy
< date: Tue, 12 Sep 2017 15:20:53 GMT
< content-length: 0
<
* Connection #0 to host localhost left intact
```

**Comment by tommyjcarpenter** [comment]
Sorry my bad. I didn't `rebar3 upgrade`. I can confirm this fixes it. Thank you!

I brought this into my own fork which upgrades the jiffy version.

File: rebar.config [hunk 1]
```diff
@@ -1,10 +1,7 @@
-%%-*- mode: erlang -*-
+{require_min_otp_vsn, "18"}.
 
-{deps, [
-        {cowboy, ".*", {git, "git://github.com/extend/cowboy.git", {tag, "0.9.0"}}},
-        {jiffy, ".*", {git, "git://github.com/davisp/jiffy.git", {tag, "0.8.5"}}},
```
Comment (tommyjcarpenter) [inline rebar.config]: Naive question.. what is the effect of removing jiffy here? I am using a combination of leptus and Jiffy in a few projects; is Jiffy completely interchangeable with msgpack, such that jiffy + leptus 1.0.0 (your proposal here) is fine?

Comment (lukyanov) [reply rebar.config]: Sorry. I didn't meant this PR be such big. It is actually my mistake as I initially made a PR from my master branch. Now I added a lot of stuff to my master and it all appeared here. I'm going to close this PR.

As regards to your question, jiffy has been removed as an alternative to jsx (https://github.com/talentdeficit/jsx.git). leptus originally supported two different JSON libraries, in my changes it supports only one (jsx). msgpack is still supported as before.

**Pull request closed without merging**